carp.common/src/commonMain/kotlin/dk/cachet/carp/common/application/devices/DeviceRegistration.kt

```diff
@@ -28,7 +29,7 @@ abstract class DeviceRegistration
      */
     abstract val deviceId: String
 
-    val registrationCreationDate: DateTime = DateTime.now()
+    val registrationCreationDate: Instant = Clock.System.now()
```

Comment:
Same as prev comment. I know this was the old name as well but for v1 I believe this should be renamed to reflect its a complete timestamp